AccountServiceTests/ServiceTests/AccountServiceTest.cs

```diff
@@ -0,0 +1,342 @@
+﻿using System;
+using System.Text;
+using System.Threading.Tasks;
+using AccountService.Domain;
+using AccountService.Helpers;
+using AccountService.Models;
+using AccountService.Repositories;
+using AccountService.Services;
+using Microsoft.AspNetCore.Mvc.ModelBinding.Metadata;
+using Moq;
+using NuGet.Frameworks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace AccountServiceTests.ServiceTests
+{
+    public class AccountServiceTest
+    {
+        private readonly IAccountService _accountService;
+        private readonly Mock<IAccountRepository> _repository;
+        private readonly Mock<IHasher> _hasher;
+        private readonly Mock<IJWTokenGenerator> _jwtGenerator;
+        private readonly Mock<IRegexHelper> _regexHelper;
+
+
+        public AccountServiceTest()
+        {
+            _jwtGenerator = new Mock<IJWTokenGenerator>();
+            _hasher = new Mock<IHasher>();
+            _repository = new Mock<IAccountRepository>();
+            _regexHelper = new Mock<IRegexHelper>();
+            _accountService = new AccountService.Services.AccountService(_repository.Object, _hasher.Object, _jwtGenerator.Object,
```

Comment:
Plaats een enter na AccountService( zodat alles fatsoenlijk leesbaar is en haal hieronder de whitespacing weg